lib/main_common.pm

```diff
@@ -471,7 +471,10 @@ sub load_zdup_tests {
     # Restrict version switch to sle until opensuse adopts it
     loadtest "migration/version_switch_upgrade_target" if is_sle and get_var("UPGRADE_TARGET_VERSION");
     loadtest 'boot/boot_to_desktop';
-    loadtest 'console/check_upgraded_service' if !is_desktop;
+    if (!is_desktop) {
```

Comment:
Is there any reason that we can't run the check on SLED?

Reply:
valid question , also I would ask to enter answer to this question as comment here . see line 471 as example